.github/workflows/ci.yml

```diff
@@ -85,8 +85,9 @@ jobs:
         env:
           ROLLBAR_ACCESS_TOKEN: ${{ secrets.ROLLBAR_ACCESS_TOKEN }}
       - name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v2
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
I was about to ask why v2 didn't work but it appears v1 is the latest/only published version 😆